PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Throwing a generic 'Test Exception' indicates leftover debugging code. This should be removed or replaced with a meaningful error condition.

 **SUGGESTION:**

```suggestion
 // remove or replace with actual error handling
```